src/gridstack-dd.ts

```diff
@@ -449,6 +449,15 @@ GridStack.prototype._prepareDragDropByNode = function(node: GridStackNode): Grid
     this.engine.moveNode(node, x, y, w, h);
     if (resizing && node.subGrid) { (node.subGrid as GridStack).onParentResize(); }
     this._updateContainerHeight();
+
+    if (this._gsEventHandler[event.type]) {

```

Comment:
looks like we should also add 'drag' event... also we need to consider if we want to call it for pixel changes, or just when column/row changes happen (what you have)

Right now you have it if only column/row changes and passing node copy (why clone the values ? just pass the node as it has already moved to new location...
I would pass node.el instead to be consistent with start/top events AND add drag event as well.

Reply:
I had a problem getting the Element with the updated values. I've overlooked the node.el, this does the trick.

I added it to column/row changes, because I think it makes the most sense regarding snapping. The moment it snaps to another grid you want an update and do something with that

Reply:
I've changed it to use the target, but I had to use ``_writePosAttr`` to get the current attributes while resizing